test/lib/recordevents/resources.go

```diff
@@ -133,7 +133,7 @@ func recordEventsPod(imageName string, name string, serviceAccountName string) *
 			Containers: []corev1.Container{{
 				Name:            imageName,
 				Image:           pkgtest.ImagePath(imageName),
-				ImagePullPolicy: corev1.PullIfNotPresent,
+				ImagePullPolicy: corev1.PullAlways,
```

Comment:
Is there a reason for changing this, I think this is going to cause issues with kind based tests.

Reply:
Sorry, i commited this by mistake :smile: